src/webviews/extension-side/webviewHost.ts

```diff
@@ -161,10 +159,6 @@ export abstract class WebviewHost<IMapping> implements IDisposable {
                 this.webViewRendered();
                 break;
 
-            case CssMessages.GetCssRequest:
```

Comment:
I'd rename this to `GetTheme` which would return `isDark`

Reply:
Thank you! can do!